src/helm/benchmark/presentation/create_plots.py

```diff
@@ -0,0 +1,583 @@
+
+        method_to_label = {
+            "multiple_choice_joint": "Multiple Choice Joint",
+            "multiple_choice_separate_original": "Multiple Choice Separate",
+            "multiple_choice_separate_calibrated": "Multiple Choice Separate Calibrated",
+        }
+        palette = get_color_palette(len(method_to_label))
+        width = 0.2
+        for i, column in enumerate(table.columns):
+            ax = axarr[i // num_columns][i % num_columns]
+            for j, method in enumerate(method_to_label):
+                models: List[str] = []
+                ys: List[float] = []
+                for adapter, accuracy in zip(table.adapters, column.values):
+                    if method not in adapter:
+                        continue
+                    models.append(adapter.split(" [")[0])
+                    ys.append(accuracy)
+                xs = np.arange(len(models))
+                ax.bar(xs + (j - 1) * width, ys, width, color=palette[j], label=method_to_label[method])
+                ax.set_xticks(xs, models, rotation=-20, ha="left")
+                ax.set_title(f"column.group ({column.metric})")
+                if i == 0:
+                    ax.legend(ncol=3, loc="upper left", bbox_to_anchor=(0, 1.4))
+        fig.subplots_adjust(wspace=0.25, hspace=0.65)
+        self.save_figure(fig, "mc_ablations.pdf")
+
+    def create_constrast_set_plots(self):
+        """For each contrast set scenario, plot the accuracy and robustness of each model on a scatter plot."""
+        tables = self.get_group_tables("robustness_contrast_sets")
+        fig, axarr = plt.subplots(1, 2, figsize=(7, 3))
+        for ax, (table_name, table) in zip(axarr, tables.items()):
+            xs = [column for column in table.columns if column.name == "EM"][0].values
+            ys = [column for column in table.columns if column.name == "EM (Robustness)"][0].values
+            ax.scatter(xs, ys)
+            ax.plot([0, 1], [0, 1], color="gray", ls="--")
+            ax.set_title(table.columns[0].group)
+            ax.set_xlabel("Accuracy")
+            ax.set_ylabel("Robustness")
+        fig.subplots_adjust(wspace=0.25)
+        self.save_figure(fig, "contrast_sets.pdf")
+
+    def create_all_plots(self):
+        """Create all the plots used in the HELM paper."""
+        self.create_accuracy_v_x_plots()
+        self.create_correlation_plots()
+        self.create_leaderboard_plots()
+        self.create_all_accuracy_v_model_property_plots()
+        self.create_accuracy_v_access_bar_plot()
+        self.create_task_summary_plots()
+        for targeted_eval in ["language", "knowledge", "reasoning"]:
+            self.create_targeted_eval_plot(targeted_eval)
+        self.create_copyright_plot()
+        self.create_bbq_plot()
+        self.create_in_context_examples_plot()
+        self.create_mc_ablations_plot()
+        self.create_constrast_set_plots()
+
+
+def main():
```

Comment:
Can we put instructions on how to run this script along with a bit more documentation that says this is for the HELM paper (link to arxiv).